docs/docs/tutorials/get-started/README.mdx

```diff
@@ -138,6 +138,17 @@ You need to set the environment variable `TRUST_PROXY_HEADER` to `true`. See [Co
 
 </details>
 
+<details>
+
+<summary>
+  I can load Admin Console now，and have successfully created an admin account. But when I tried to
+  login, it failed with error: <code>TypeError: Failed to fetch</code>
+</summary>
+
+You need to set `X-Forward-Proto` header to `https` in your server config. See [Trusting TLS offloading proxies](https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#trusting-tls-offloading-proxies) for details.
```

Comment:
can this combine with the upper section? or they are two different things? sry i'm sleep the question may be dumb

Reply:
Haha. No, I think they are two independent problems. The upper one requires to set a node env variable, whereas this one requires to set an additional header in the nginx server configs.

Reply:
Discussed offline and have merged the last two questions into one. PTAL
<img width="846" alt="image" src="https://user-images.githubusercontent.com/12833674/180725855-4b7539e0-06b4-494f-821b-3df687aa7286.png">